plugins/modules/dcnm_links.py

```diff
@@ -0,0 +1,3326 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2022 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+__author__ = "Mallik Mudigonda"
+
+DOCUMENTATION = """
+---
+module: dcnm_links
+short_description: DCNM ansible module for managing Links.
+version_added: "2.1.0"
+description:
+    - DCNM ansible module for creating, modifying, deleting and querying Links
```

Comment:
Let's check with Shangxin but I think we need to start using NDFC instead of DCNM as the module names and documentation references.

Reply:
Sure. Let me know. I can change it

Reply:
Before we make any changes though we need to check if we can alias based on the old name.

https://ansible-docs.readthedocs.io/zh/stable-2.0/rst/developing_modules.html#deprecating-and-making-module-aliases